src/components/RequestBox/RequestBox.js

```diff
@@ -378,10 +379,12 @@ export default class RequestBox extends Component {
       },
     });
 
+    let medicationRequestReference = medicationDispense.authorizingPrescription[0].reference;
+
     this.setState({ gatherCount: 1 });
     client
       .request(`MedicationDispense/${medicationDispense.id}`, {
-        resolveReferences: ["performer.0.actor"],
+        resolveReferences: ["performer.0.actor", "authorizingPrescription.0"],
```

Comment:
What was this reference added for?  I can't seem to figure it out.

Reply:
MedicationDispense https://www.hl7.org/fhir/medicationdispense.html doesn't have insurance information as it is for medication refill.  CRD needs the payor info to search for the CRD rule. This reference is used to find the payor info from the MedicationRequest the MedicationDispense is referring to.

Reply:
Ok.  It seems like this particular reference isn't used in the code anywhere, and instead a workaround was added?

Reply:
Yes, this is the case.